README.md

```diff
@@ -1,4 +1,27 @@
+## Introduction
 
+**What?** The [Neurodata Without Borders: Neurophysiology][nwb-neurophysiology] (NWB:N) Hackathon is a week-long hands
+on activity in which neurophysiology researchers create solutions using the open source NWB:N software packages.
+Participants work collaboratively on solutions to develop a unified data format for cellular-based neurophysiology data, focused on the dynamics of groups of neurons measured under a large range of experimental conditions. In contrast to conferences and workshops where the primary focus is to report results, the objective of the Hackathon is to provide a venue for creators of neurophysiology open-source software to collaboratively work.
+
+[nwb-neurophysiology]: http://www.nwb.org/nwb-neurophysiology/
+
+
+**When, where, how much?** Twice a year - Early April in Seattle, and late April in Berkley. Ad-hoc meetings are added occasionally. There is no registration fee, and lunch and dinner are usually covered.
+
+**How does it work?** Weekly videoconferences for preparation begin 8-12 weeks before an event. Potential participants propose projects during these meetings, and collaboratively create a list of projects that are of mutual interest. The projects include platform work, algorithm development, and neurophysiology applications. Through the course of the meetings, each participant selects one or more project teams and develops goals for the week. The first day of the Project Week itself begins with a 2-hour in-person introduction to all projects and participants. The rest of the week consists of a mix of working sessions and breakout sessions on special topics, as decided by the participants during the preparatory meetings.
+
+**Who can attend?** Hackathons are open to XXXX and publicly advertised. One-day participation is permitted for first-time attendees, but rest stay for the entire event. Email announcements are sent to the [Hackathon mailing list](To be done) (for continuity) and posted on the [Hackathon forum](https://github.com/orgs/NeurodataWithoutBorders/teams/hck01/discussions).
```

Comment:
but rest stay for the entire event => but participants are strongly encouraged to stay for the entire event